projects/storefrontlib/src/cms-components/myaccount/order/replenishment-order-details/replenishment-order-details-order-history/replenishment-order-details-order-history.component.html

```diff
@@ -0,0 +1,155 @@
+<ng-container *ngIf="orders$ | async as orderHistory">
```

Comment:
So I've gone about comparing this template with `order-history.component.html`. It's not easy since you renamed orders to orderHistory and removed a wrapping div. 

What I want to know is, what are the differences between this html and `order-history.component.html`?

Reply:
4 differences

1 - rename orders to orderHistory for better readability
2 - need to remove that wrap because the container making the template too small, whereas it needs to be the same width as the other components in the page
3 - the different on what is shown for when there is no orders. Instead of the button + text, there's only text
4 - the styling needs to be different

all in all, I believe there's no way to modify the styling if I were to literally re-use as it will touch the other order history component page and the display for no content

Reply:
-----------
as discussed uses composition instead of inheritance